sources/api/thar-be-settings/src/service.rs

```diff
@@ -65,12 +113,12 @@ where
 /// Given a map of Setting to affected Service name, return a
 /// HashSet of affected service names
 fn get_affected_service_names(
```

Comment:
I just realized this whole function can be replaced by a simple iterator function call :)

So above, we should be able to do this, and then remove the function entirely.
```diff
-let service_names = get_affected_service_names(&affected_services);
+let service_names = affected_services.values().flatten().collect();
```